### DISCUSSION TITLE
Synchronizing a Microsoft Teams folder (or file) through OneDrive for Business

### DISCUSSION THREAD

**Discussion by pablob127** [abraunegg/onedrive#2412] [Q&A] [answered]
Hello,

At my work, we have OneDrive for Business, and we typically use Teams to share folders with documents. Based on Microsoft's documentation, it is possible to sync Teams folders with OneDrive by opening them in Sharepoint and clicking "Sync". When I do this, it looks like the browser tries to open some strange URI, which I guess is interpreted by the Windows OneDrive application, and does not have any OneDrive API equivalent.

Would anyone have any suggestion on how I could make OneDrive for Linux to sync those folders?

I tried to list the "Shared Business Folders", but those are not there.

Thanks!

**Comment by abraunegg** [comment, marked as answer]
@pablob127 
The functionality is unique to using a GUI - and this is a CLI tool. Please refer to: 

https://github.com/abraunegg/onedrive/issues/2389

https://github.com/bpozdena/OneDriveGUI/issues/101

Additionaly the OneDrive API does not support this.

**Reply by pablob127** [reply]
Thank you very much!
